### ISSUE TITLE
Unhandled search key event <...>

### ISSUE THREAD

**Issue by rameshdharan** [walles/moar#20]
Trying out moar on my macOS Catalina workstation. I reproed this both with HEAD and the 0.9.17 official release: 

1. export PAGER=moar
2. man bash
3. hit '/', then ctrl-c, ctrl-r, other keypresses that don't do anything

After quitting I see in my terminal, e.g.

```
$ man bash
Please post the following report at <https://github.com/walles/moar/issues>,
or e-mail it to johan.walles@gmail.com.

Version: 0.9.17
LANG   : en_US.UTF-8
TERM   : xterm-256color

GOOS    : darwin
GOARCH  : amd64
Compiler: gc
NumCPU  : 4

2019/12/06 00:25:56 Unhandled key event 18
2019/12/06 00:25:57 Unhandled rune keypress 'l'
2019/12/06 00:25:57 Unhandled rune keypress 's'
2019/12/06 00:25:57 Unhandled rune keypress 's'
2019/12/06 00:25:58 Unhandled key event 3
2019/12/06 00:25:58 Unhandled key event 18
```

**Comment by rameshdharan** [comment]
As a workaround of course I can do:

`export PAGER='moar 2>/dev/null'`

But I'd prefer having this be the default and maybe only printing that stuff with --verbose or similar?

**Comment by walles** [comment]
A `--debug` flag sounds like a plan, I'll look at implementing something using <https://github.com/sirupsen/logrus>.

**Comment by walles** [comment]
Fix released in 0.9.18, please try it out @rameshdharan and let me know if it helped!
